scripts/hal_alignment/hal_gene_liftover.py

```diff
@@ -58,7 +58,12 @@ def make_src_region_file(regions: Iterable[Union[pybedtools.cbedtools.Interval,
         bed_file: Path of BED file to output.
         flank_length: Length of upstream/downstream flanking regions to request.
 
+    ValueError: If the flank length is negative.
```

Comment:
Following Google docstring format:
```suggestion
    Raises:
        ValueError: If `flank_length` is negative.
```